cloud-stream-kafka/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+services:
+  kafka:
+    image: wurstmeister/kafka
+    ports:
+      - "9092:9092"
+    environment:
+      - KAFKA_ADVERTISED_HOST_NAME=127.0.0.1
+      - KAFKA_ADVERTISED_PORT=9092
+      - KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181
+    depends_on:
+      - zookeeper
+  zookeeper:
+    image: wurstmeister/zookeeper
+    ports:
+      - "2181:2181"
```

Comment:
Please us the single port notation - this will prevent port conflicts on the CI nodes. The other kafka examples already do this, you can take a look there.